ui/js/directives/treeherder/clonejobs.js

```diff
@@ -558,19 +557,28 @@ treeherder.directive('thCloneJobs', [
                             countBtnHTML += element;
                         });
                     }
+                    var display_style = "none";
+                    if (btnHTML.indexOf('filter-shown') != -1 || countBtnHTML.indexOf('filter-shown') != -1) {
+                        display_style = "inline";
+                    }
                     jobTdHtml += jobGroupInterpolator({
                         btnHTML: btnHTML,
                         countBtnHTML: countBtnHTML,
                         symbol: jobGroup.symbol,
                         name: jobGroup.name,
-                        grkey: jobGroup.grkey
+                        grkey: jobGroup.grkey,
+                        display: display_style
```

Comment:
You could actually do something like:
display: (btnHTML.indexOf('filter-shown') !== -1 || countBtnHTML.indexOf('filter-shown') !== -1) ? 'inline' : 'none'